docs/basics/acct-team-mgmt/concurrency/understanding-concurrency.md

```diff
@@ -0,0 +1,72 @@
+---
+id: understanding-concurrency
+title: Understanding Concurrency
+sidebar_label: Understanding Concurrency
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<!-- <p><span className="sauceGreen">Enterprise Plans only</span></p> -->
+
+Your subscription to the Sauce Labs Virtual Cloud or Real Device Cloud entitles you to run a certain amount of concurrent tests as many times as you like within a given region. In essence, this means you pay for the ability to run at your peak needs and you can run at that as frequently as you like. Your concurrency can be used for both automated and live testing.
+
+## Virtual Cloud Concurrency
+
+Concurrency for the virtual cloud is defined at two levels, the organization and the team level. 
+
+To view your organizations concurrency limit:
+
+1. In Sauce Labs, click **ACCOUNT** and then click **Organization Management**.
+
+<img src={useBaseUrl('img/team-mgmt/team-mgmt-nav.png')} alt="Organization management navigation" width="300"/>
+
+2. On the **Organization Management** page, under the ORGANIZATION NAME box, concurrency limits will be displayed for the selected data center. 
+
+3. To view the limits for another data center, in the upper-right corner of the window, click the **DATA CENTER** dropdown and select the relevant data center.
+
+<img src={useBaseUrl('img/team-mgmt/ccy-data-center-dropdown.png')} alt="Data Center dropdown" width="250"/>
+
+To view the concurrency for a given team
+
+4. On the **Organization Management** page, click the **TEAMS** tab. **Team VM Concurrency** and **Peak VM Concurrency** will be displayed for each team in the organization.
+
+### Exceeding Virtual Cloud Concurrency
+
+The virtual cloud is used to run business critical pipelines across large enterprise organizations. This means there will be times when it is difficult to predict exactly how much concurrency you will need at all times. For these situations we allow your organization to exceed it's concurrency limit. 
```

Comment:
"virtual cloud" or `Virtual Cloud`? There are instances, also in this document, where we refer to RDC as `Real Device Cloud`. Not sure what our standard casing is for these terms, but should be consistent at least within a doc 🤔